### ISSUE TITLE
ng build fails to copy .htaccess, and possibly other similarly named files, to the output

### ISSUE THREAD

**Issue by MrCroft** [angular/angular-cli#1758]
`.htaccess` file from `public` folder isn't copied over to `dist` on `ng build`

```
angular-cli: 1.0.0-beta.11-webpack.2
node: 6.4.0
os: win32 x64
```

**Comment by hansl** [comment]
@TheLarkInn Are you not copying dot files over?

**Comment by MrCroft** [comment]
My `.htaccess` file from the `public` folder still isn't being copied over to `dist` on build.
In addition to that, now other files (from the same `public` folder) aren't copied as well. For example: `favicon.ico`, `favicon.png`, `test.json`
Only `.npmignore` is, but it puts it in an `assets` folder, instead of keeping the structure (I would have expected to copy `.npmignore` in an `assets` folder only if it would have been in an `assets` folder in `public` also, but it is directly in the `public` folder.
I don't know if that's the plan here, but if it is... even if some of the files could be in a subfolder, other files like `.htaccess` have to be in the `dist` root.

```
angular-cli: 1.0.0-beta.11-webpack.8
node: 6.4.0
os: win32 x64
```

**Comment by chateaux** [comment]
I was having this issue as well and then I realised that I am just being stupid.

I was initially running the build process manually and copying a zip of the distribution to the server and unzipping it. Prior to the zip I would copy any other files I needed into the distribution. 

This is/was a pain to do.

However, once I have completed testing something, it gets moved into a git/jenkins scenario which makes life very simple.

This is my workflow:

1. Commit to github
2. Jenkins is notified and pulls the code (obviously not develop code)
3. Jenkins compiles the code for distribution (ng build --prod --aot)
4. Jenkins copies to the dist folder all my required files (htaccess, robots, etc.)
5. Jenkins scp's a tar of the dist to my server.
6. Jenkins deletes server files and untars the new environment.

Takes at most two minutes from start to end and to initiate the process is pretty much a: git push upstream master from my local command line.
